src/api/alerts.rs

```diff
@@ -1,74 +1,88 @@
-
-use std::ops::BitOrAssign;
-
+use crate::models::alert_models;
 use actix_web::{get, web, HttpResponse};
-use mongodb::{bson::{doc, Document}, Client, Collection};
 use futures::TryStreamExt;
-use crate::models::alert_models;
+use mongodb::{
+    bson::{doc, Document},
+    Client, Collection,
+};
 
 const DB_NAME: &str = "boom";
 
 // Get an object from a particular catalog by using its objectid
 #[get("/alerts/get_object")]
-pub async fn get_object(client: web::Data<Client>, body: web::Json<alert_models::GetObjectBody>) -> HttpResponse {
+pub async fn get_object(
+    client: web::Data<Client>,
+    body: web::Json<alert_models::GetObjectBody>,
+) -> HttpResponse {
     let body = body.clone();
     let catalog = body.catalog.expect("catalog required for get_object");
-    let collection: Collection<mongodb::bson::Document> = 
-        client.database(DB_NAME).collection(&format!("{}_alerts", catalog));
+    let collection: Collection<mongodb::bson::Document> = client
+        .database(DB_NAME)
+        .collection(&format!("{}_alerts", catalog));
     let object_id = body.object_id.expect("objectId required for get_object");
 
     // get brightest alert
```

Comment:
Since we are trying to find the brightest one, instead of fetching all of them for an object and then looping over it to find which has the lowest magpsf value (so the brightest), maybe we can simply sort by magpsf ascending (lowest first, i.e. the brightest) and only return one? That way no need for fetching a lot of data from the DB. Some objects could have thousands of alerts